scss/civicrm/contact/pages/_export.scss

```diff
@@ -146,3 +146,29 @@
     }
   }
 }
+
+.crm-export-ui-save-mapping-dialog {
+  .crm-form-block {
+    box-shadow: none;
+  }
+
+  .ui-dialog-buttonpane.ui-widget-content {
+    background: none;
+    border: 0;
+    box-shadow: none;
+    float: left;
+    height: auto;
+    margin-top: 30px;
+    padding-bottom: 0;
+    padding-left: 0;
+    padding-right: 0;
+
+    .crm-button.ng-binding {
```

Comment:
`ng-binding` is a class added by angularjs. It should not be used.

Reply:
Removed (I force-pushed the changes then is all in the same commit)